libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/headerSection.spec.ts

```diff
@@ -72,25 +72,37 @@ describe("Header section", () => {
             insightCatalog.waitForCatalogReload();
         });
 
-        it("(SEPARATE) limitation of title, description", () => {
-            cy.fixture("headerDataTest").then((data) => {
-                const title = data["LimitTexts"].title;
-                const desc = data["LimitTexts"].description;
-                const headerRow_01 = layoutRow_01.getHeader();
-                const headerRow_02 = layoutRow_02.getHeader();
-                insightCatalog.waitForCatalogReload();
-                headerRow_01.setTitle(title).selectTitleInput().hasLimitMessage("128/256 caractères restant");
-                headerRow_02
-                    .scrollIntoView()
-                    .setDescription(desc)
-                    .selectDescriptionInput()
-                    .hasLimitMessage("512/1024 caractères restant");
+        //Cover ticket: RAIL-4674
+        it(
+            "(SEPARATE) limitation of title, description",
```

Comment:
(SEPARATE) title used on for isolated test case. If test run with integrated, you can remove it